Tests/EndToEndTests/CNTKv2Python/Tutorials/CNTK_302B_Image_Super-resolution_Using_CNNs_and_GANs_test.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+# Licensed under the MIT license. See LICENSE.md file in the project root
+# for full license information.
+# ==============================================================================
+
+import os
+import re
+import numpy as np
+
+abs_path = os.path.dirname(os.path.abspath(__file__))
+notebook = os.path.join(abs_path, "..", "..", "..", "..", "Tutorials", "CNTK_302B_Image_Super-resolution_Using_CNNs_and_GANs.ipynb")
+datadir = os.path.join(abs_path, "..", "..", "..", "..", "Examples", "Image", "DataSets", "BerkeleySegmentationDataset")
+
+# Run this on GPU only
+notebook_deviceIdsToRun = [0]
+
+@pytest.fixture(scope='module')
```

Comment:
comment this out

Reply:
notebook_deviceIdsToRun = [0]
notebook_timeoutSeconds = 1800

#comment this out
#@pytest.fixture(scope='module')